api_reflector/admin.py

```diff
@@ -97,6 +113,11 @@ class ResponseView(RestrictedView):
     form_widget_args = {"content": {"rows": 8, "style": "font-family: monospace;"}}
     column_searchable_list = ("name",)
 
+    def on_model_change(self, form, model, is_created):
+        if validator := form_validators.get(form.content_type.data):
```

Comment:
AC2 on the ticket is not very clear. Currently the QA and BA's often leave the content field empty because they don't need the initial response on a join journey. However, this would not be a valid json response and would raise an error when the check runs. To make things easier please can you put a default empty json message in the contents {}.